types/schema/data/Proposal.ts

```diff
@@ -49,84 +35,117 @@ export interface BaseDetails {
-      count: {
-        existing: number;
-        proposed: number;
-      };
-      onStreet?: {
-        count: {
-          existing: number;
-          proposed: number;
-        };
-        club?: VehicleParkingCount;
-        disabled?: VehicleParkingCount;
-        other?: VehicleParkingCount;
-        residents?: VehicleParkingCount;
-      };
-      offStreet?: {
-        count: {
-          existing: number;
-          proposed: number;
-        };
-        club?: VehicleParkingCount;
-        disabled?: VehicleParkingCount;
-        other?: VehicleParkingCount;
-        residents?: VehicleParkingCount;
-      };
-    };
-    vans?: {
-      count: {
-        existing: number;
-        proposed: number;
-      };
-      onStreet?: VehicleParkingCount;
-      offStreet?: VehicleParkingCount;
-    };
-    motorcycles?: {
-      count: {
-        existing: number;
-        proposed: number;
-      };
-      onStreet?: VehicleParkingCount;
-      offStreet?: VehicleParkingCount;
-    };
-    bicycles?: {
-      count: {
-        existing: number;
-        proposed: number;
-      };
-      onStreet?: VehicleParkingCount;
-      offStreet?: VehicleParkingCount;
-    };
-    buses?: {
-      count: {
-        existing: number;
-        proposed: number;
-      };
-      onStreet?: VehicleParkingCount;
-      offStreet?: VehicleParkingCount;
-    };
+export interface LondonProposal extends BaseProposal {
+  schemeName?: string;
+  /**
+   * @description Proposed parking spaces
+   */
+  parking?: {
+    cars?: ProposedCount;
+    vans?: ProposedCount;
+    motorcycles?: ProposedCount;
+    cycles?: ProposedCount;
+    buses?: ProposedCount;
+    disabled?: ProposedCount;
+    carClub?: ProposedCount;
+    offStreet?: {residential: ProposedCount};
+    other?: ProposedCount;
+  };
+  /**
+   * @description Creating new buildings
+   */
+  newBuildings?: NewBuildingsOrStoreys;
+  /**
+   * @description Increasing the height of existing buildings
+   */
+  newStoreys?: NewBuildingsOrStoreys;
+  /**
+   * @description Project cost
+   */
+  cost?: {
+    projected: '2m' | '2mTo100m' | '100m';
+  };
+  /**
+   * @description Electric vehicle charing points
+   */
+  charging?: {
+    active: {count: number};
+    passive: {count: number};
+  };
+  /**
+   * @descrption Changes that result in the loss, gain, or change of use of natural spaces
+   */
+  nature?: {
+    openSpaces?: {
+      impact: 'loss' | 'gain' | 'change';
+      description: string;
+      // @todo add type & designation from enums/Nature
+      access: 'restricted' | 'unrestricted';
+      area: {hectares: number};
+      /**
+       * @description Whether the open space change involves a land swap
+       */
+      swap: boolean;
+    }[];
+    protectedSpaces?: {
+      impact: 'loss' | 'gain' | 'change';
+      description: string;
+      // @todo add designation from enums/Nature
+      access: 'restricted' | 'unrestricted';
+      area: {hectares: number};
+    }[];
```

Comment:
`openSpaces` & `protectedSpaces` are examples of types of data we expect to capture in the upcoming list component. 

Eg in content these will look like `proposal.nature.openSpace.one.impact`, `proposal.nature.openSpace.two.impact` and so on. I'm intentionally omitting the "one", "two" etc from the schema types in favor of an array, we'll just handle the index text during passport mapping!